src/components/About/ProfileCard.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+
+// styling
+import styles from "./ProfileCard.module.css";
+import Typography from "../utility/Typography/Typography";
+
+// images
+import crossImg from "../../assets/icon-cross.svg";
+import closeImg from "../../assets/icon-close.svg";
+import twitterImg from "../../assets/icon-twitter.svg";
+import linkedinImg from "../../assets/icon-linkedin.svg";
+
+const ProfileCard = ({ person }) => {
+  const { name, title, quote, avatar } = person;
+  const [showQuote, setShowQuote] = useState(false);
+
+  const handleClick = () => {
+    setShowQuote(!showQuote);
+  };
+
+  return (
+    <div>
+      <div className={styles.container}>
+        {!showQuote ? (
+          <>
+            <img src={avatar} alt={`${name}`} className={styles.avatar} />
+            <Typography elType="h3" className={styles.name}>
+              {name}
+            </Typography>
+            <Typography elType="body2" className={styles.title}>
+              {title}
+            </Typography>
+            <button className={styles.button} id="show" onClick={handleClick}>
```

Comment:
Do not use `id` here. This component will be on the page multiple times, and an `id` should appear on the page exactly once.

Reply:
I did not know that! Thanks!